tensorflow_addons/image/median_filter_2D_test.py

```diff
@@ -0,0 +1,102 @@
+from tensorflow.python.platform import test
+from tensorflow.python.framework import ops
+from tensorflow.python.framework import dtypes
+from tensorflow.python.ops import array_ops
+import median_filter_2D as md
+
+import numpy as np
+
+
+class Median2DTest(test.TestCase):
+
+
+  def _validateMedian_2d(self, inputs, expected_values, filter_shape = (3, 3)):
+    np_expected_values = np.array(expected_values)
+    values_op = md.median_filter_2D(inputs)
+    with self.test_session(use_gpu=False) as sess:
+      values = values_op.eval()
+      self.assertShapeEqual(values, inputs)
+      self.assertShapeEqual(np_expected_values, values_op)
+      self.assertAllClose(np_expected_values, values)
+
+
+  def testfiltertuple(self):
+    tf_img = np.zeros((3, 4, 3))
+    tf_img = ops.convert_to_tensor(tf_img)
```

Comment:
how about using tf.zeros to create a tensor and remove dependency on np.

Reply:
solved